gui/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ qt5_wrap_ui(euRun_UISrcs ui/euRun.ui)
 qt5_wrap_cpp(euRun_MOCSrcs include/RunControlModel.hh include/euRun.hh)
 list(APPEND RUN_GUI_SRC ${euRun_UISrcs} ${euRun_MOCSrcs})
 add_executable(euRun ${RUN_GUI_SRC})
-target_link_libraries(euRun ${EUDAQ_CORE_LIBRARY} ${EXTERNAL_LIB_QT})
+target_link_libraries(euRun ${EUDAQ_CORE_LIBRARY} ${EXTERNAL_LIB_QT} ${EUDAQ_CORE_LIBRARY})
```

Comment:
linking same lib twice

Reply:
That was deliberate as there was an issue on Windows. The fixes I will move into a separate MR...it will likely re-appear.

Reply:
But removed for the moment.